docs/access.md

```diff
@@ -1,20 +1,118 @@
 # Access management
 
-## LaminDB
+This document outlines the access control system, detailing its architecture, key features, roles, and implementation guidelines.
 
-LaminDB doesn't come with access management but relies on the underlying data infrastructure for this.
+## System architecture
```

Comment:
Please use "Concepts" or "Entities". What you're describing in this section is not a system architecture.